### PR TITLE
fix(unchanged): Remnant Side Missions Fix

### PR DESCRIPTION
**Pull request by TheGiraffe3**
**Bug fix**

This PR addresses the bug/feature described in issue 
None

## Summary
There were a few problems in the `Remnant: Will Not Someone Please Think Of The Children` mission. I fixed them. 

## Screenshots
None. 

## Usage examples
N/A

## Testing Done
None. 

## Save File
This save file can be used to test these changes:
None. I found this while looking through merged PRs. 

## Artwork Checklist
N/A

## Wiki Update
N/A

## Performance Impact
I sure hope not.

### CODE REVIEW

**Comment by TheGiraffe3** [comment]
This is a rather high-priority PR for me, because part of it is a bad typo fix, so I’m pretty sure it should be in 0.10.8.

**Comment by TheGiraffe3** [comment]
@Amazinite will you merge this before 0.10.8?

**Comment by Saugia** [comment]
This cut it pretty close to release time, being a day before it. As it's not a strict bug or broken code change, t's fine if it comes with the next update.

**Comment by TheGiraffe3** [comment]
Well, it’s mentioning something whose name was changed. I’m sorry because I just noticed it, but it’s rather important.

**Comment by Saugia** [comment]
We're already going through the release process and running workflows and merged the changelog; it will have to be next update.

On a side note, the name is not *entirely* incorrect, the prior name is still technically the ship's name.

**Comment by TheGiraffe3** [comment]
Players don’t know that, but okay.

**Comment by Saugia** [comment]
A typo like this is not unheard of; we find typos that can be confusing all the time; this isn't any more important than those. No need to rush it. It comes down to this being opened too close to release; it happens, and it won't hurt gameplay or anything, and will still be fixed anyways.

File: data/remnant/remnant 2 side missions.txt [hunk 1]
```diff
@@ -761,11 +761,11 @@ mission "Remnant: Will Not Someone Please Think Of The Children"
 		dialog `The Remnant are puzzled at your aborting of this mission, but they tell you they will send someone else to do this work. They warn you they will not propose any more of the sort to you again.`
 	on fail
 		"reputation: Remnant" -= 250
-		dialog `You were unable to disable the Dathnak A'awoj without destroying it... The Remnant will not be pleased...`
+		dialog `You were unable to disable the Dathnak A'awoj without destroying it... the Remnant will not be pleased...`
```
Comment (tibetiroka) [inline data/remnant/remnant 2 side missions.txt:764]:
```suggestion
		dialog `You were unable to disable the Dathnak A'awoj without destroying it... The Remnant will not be pleased...`
```

**Comment by TheGiraffe3** [comment]
This PR is ready for review/merge.

**Pull request merged**